### DISCUSSION TITLE
Can't boot on Tanix W2..

### DISCUSSION THREAD

**Discussion by Bendy3192** [devmfc/debian-on-amlogic#23] [Q&A]
Hey! Thanks for the work done. Now I'm trying to boot on my amlogic s905w2, but i see only android tv rom logo, just stuck on it. Maybe I'm doing something wrong,.?

My steps to reproduce:
1. flash .img to usb and uncomment line in boot,config
2. boot in os with usb stick, holding button in AV port

But releases also contain linux headers, sorry me for stupid questions, but what i need to do with this .deb file?

**Comment by devmfc** [comment]
Try different USB flash drive. Preferrable an older and smaller model (USB2 or older USB3). And otherwise you need to hook up a serial console to see what is going on during boot.
